server_advanced.js

```diff
@@ -12,26 +12,27 @@ const mongoose = require('mongoose');
 const app = express();
 generator.handleResponses(app, {
   predefinedSpec: function(spec) {
-    _.set(spec, 'paths["/foo/{name}"].get.parameters[0].description', 'description of a parameter');
+    _.set(spec, 'paths["/students/{id}"].get.parameters[0].description', 'description of a parameter');
     return spec;
   },
   specOutputPath: './test_spec.json',
-  mongooseModels: mongoose.modelNames()
+  mongooseModels: mongoose.modelNames(),
+  tags: ['Students']
 });
 
 app.use(bodyParser.json({}));
 let router = express.Router();
-router.route('/foo/stranger')
+router.route('/students')
```

Comment:
I think the proper path here would be `/students/stranger`. The goal of having it here is to test two pretty the same paths: `/students/stranger` vs `/students/:name`

Reply:
Got it! Just changed `server_advanced.js` example.